pkg/vm/stackitem/item.go

```diff
@@ -1019,3 +1019,60 @@ func (i *Buffer) Convert(typ Type) (Item, error) {
 func (i *Buffer) Len() int {
 	return len(i.value)
 }
+
+// DeepCopy returns new deep copy of the provided item.
+// Values of Interop and BigInteger items are not deeply copied.
```

Comment:
Why don't we copy `BigInteger`?

Reply:
I thought we always copy it in out VM opcodes, to there is no way to alter it.
But after reexamination I see, that e.g. in `NEG` we modify it directly. Fixed.